tuna/fin_eval.py

```diff
@@ -195,9 +200,9 @@ def process_fdb_eval(self, fin_json, result_str='miopen_find_eval_result'):
           fdb_entry = obj
           fdb_entry.alg_lib = fdb_obj['algorithm']
           fdb_entry.kernel_time = fdb_obj['time']
-          # workspace
           fdb_entry.workspace_sz = fdb_obj['workspace']
           fdb_entry.session = self.dbt.session.id
+          fdb_entry.params = fdb_obj['params']
```

Comment:
is session being appended anywhere to fdb_entry like: fdb_entry.session = self.session.id?

Reply:
line 204 looks like an equivalent.